src/main/java/com/indiraactive/fulfillmentplatform/viewModel/ScheduleHistoryViewModel.java

```diff
@@ -0,0 +1,37 @@
+package com.indiraactive.fulfillmentplatform.viewModel;
+
+import com.indiraactive.fulfillmentplatform.dao.supplier.SupplierRepository;
+import com.indiraactive.fulfillmentplatform.domain.ScheduledTaskHistoryModel;
+import com.indiraactive.fulfillmentplatform.dao.scheduledTask.ScheduledTaskJpa;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ScheduleHistoryViewModel {
+    private SupplierRepository supplierRepository;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'supplierRepository' could be replaced by a local variable.](https://app.codacy.com/app/indira-active/fulfillment-platform/pullRequest?prid=1507512)